tools/aerospike.prom

```diff
@@ -0,0 +1,1347 @@
+# HELP aerospike_namespace_allow_ttl_without_nsup allow ttl without nsup
+# TYPE aerospike_namespace_allow_ttl_without_nsup gauge
+aerospike_namespace_allow_ttl_without_nsup{cluster_name="null",ns="bar",service="10.0.2.15:3000"} 0
```

Comment:
is this used?

Reply:
thank you @paologallinaharbur , I have removed this file now.. 😃